R/estimateDiversity.R

```diff
@@ -296,31 +297,53 @@ setMethod("estimateDiversity", signature = c(x="TreeSummarizedExperiment"),
         }
         
         # Check tree_name
-        if( !.is_non_empty_string(tree_name) ){
-            stop("'tree_name' must be a character specifying a rowTree of 'x'.",
-                 call. = FALSE)
-        }
-        if (!is.null(assay_name)) {
-            .Deprecated(old="assay_name", new="assay.type", "Now assay_name is deprecated. Use assay.type instead.")
-        }
-        
```

Comment:
You seem to have old version. Can you please make updates to mia branches. It is much easier to review

Reply:
https://github.com/microbiome/mia/commit/aaa55f21037f29041090257d3c8ce3f397edfb3e